packages/gatsby/src/utils/worker/__tests__/jobs.ts

```diff
@@ -0,0 +1,237 @@
+import "jest-extended"
+import { createTestWorker, GatsbyTestWorkerPool } from "./test-helpers"
+import { store } from "../../../redux"
+import * as path from "path"
+import { waitUntilAllJobsComplete } from "../../jobs/manager"
+import type { MessagesFromChild, MessagesFromParent } from "../messaging"
+import { getReduxJobs, getJobsMeta } from "./test-helpers/child-for-tests"
+
+let worker: GatsbyTestWorkerPool | undefined
+
+describe(`worker (jobs)`, () => {
+  let mainProcessJobsMeta: ReturnType<typeof getJobsMeta>
+  let workersJobsMeta: Array<ReturnType<typeof getJobsMeta>>
+  let jobDescriptionsCreatedByWorkers: Array<Array<string>>
+  let mainStateAfter: ReturnType<typeof getReduxJobs>
+  let workerStateAfter: Array<ReturnType<typeof getReduxJobs>>
+  let sendMessageSpy: jest.SpyInstance<
+    void,
+    [msg: MessagesFromParent, workerId: number]
+  >
+  let receivedMessageSpy: jest.Mock<
+    void,
+    [msg: MessagesFromChild, workerId: number]
+  >
+
+  beforeAll(async () => {
+    store.dispatch({ type: `DELETE_CACHE`, cacheIsCorrupt: true })
```

Comment:
We didn't use `cacheIsCorrupt` in the other tests, why here? :)

Reply:
`jobs` reducer is specific in that it only clears its state when `cacheIsCorrupt` is set - https://github.com/gatsbyjs/gatsby/blob/53e5991c75e568934f402ae94420031d5f9f9102/packages/gatsby/src/redux/reducers/jobsv2.ts#L20-L21